### PR TITLE
Adding examples for ARF/mDL, fixes #144

### LINKED ISSUE #144: examples for ARF.
moving @tlodderstedt's and my work on adding examples how to represent PID/EAA/mDLs as SD-JWT VC to this repo from the sd-jwt repo.
EAA: https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/211
mDL: https://github.com/oauth-wg/oauth-selective-disclosure-jwt/issues/144 and https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/233
PID as a reference point: https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/304-

### PR DESCRIPTION
**Pull request by danielfett**
This adds the examples previously provided for SD-JWT to this specification, with minor adaptions.

See rendered text here: https://drafts.oauth.net/oauth-sd-jwt-vc/danielfett/arf-examples/draft-ietf-oauth-sd-jwt-vc.html#name-examples-2

Fixes #144

### CODE REVIEW

File: examples/05-arf-pid/specification.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+user_claims:
+  iss: https://pid-provider.memberstate.example.eu
+  iat: 1541493724
+  type: PersonIdentificationData
+  !sd first_name: Erika
```
Comment (paulbastian) [inline examples/05-arf-pid/specification.yml:5]:
```suggestion
  !sd given_name: Erika
```

File: examples/04-iso18013-5/specification.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+user_claims:
+  iss: "https://example.com"
+  type: "ISO_mDL"
+  docType: org.iso.18013.5.1.mDL
+  version: "1.0"
+  validityInfo:
+    signed: "2022-04-15T06:23:56Z"
+    validFrom: "2022-04-15T06:23:56Z"
+    validUntil: "2027-01-02T00:00:00Z"
```
Comment (paulbastian) [inline examples/04-iso18013-5/specification.yml:9]: Why are these not using iat, exp?

File: examples/03-eaa-diploma/specification.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+user_claims:
+  binding:
+  - !sd
+      dateOfBirth: '1973-01-01'
+      famileName: Mustermann
+      givenName: Erika
+      type: ClaimsBinding2022
+  credentialSubject:
```
Comment (paulbastian) [inline examples/03-eaa-diploma/specification.yml:8]: Whats the use of credentialSubject here?

File: examples/03-eaa-diploma/specification.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+user_claims:
+  binding:
```
Comment (paulbastian) [inline examples/03-eaa-diploma/specification.yml:2]: I expect this to be the equievalent of cnf for claim-based binding. This should be a standardized claim in the spec imo.

File: examples/03-eaa-diploma/specification.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  - !sd
+      dateOfBirth: '1973-01-01'
+      famileName: Mustermann
+      givenName: Erika
+      type: ClaimsBinding2022
```
Comment (paulbastian) [inline examples/03-eaa-diploma/specification.yml:7]:
```suggestion
      birth_date: '1973-01-01'
      family_name: Mustermann
      given_name: Erika
```

**Review by paulbastian — CHANGES_REQUESTED** [review]

**Comment by peacekeeper** [comment]
Thank you for your efforts on this. Please allow some time for participants from the EBSI community to review this.

**Comment by paulbastian** [comment]
Fyi: To my knowledge, the examples will be split into separate PRs and will undergo some changes

**Comment by peacekeeper** [comment]
In all the SD-JWT VC work, why does it sometimes use "credentialSubject" from the VCDM, and sometimes not?

**Comment by paulbastian** [comment]
In my opinion, examples with credentialSubject are deprecated

**Comment by awoie** [comment]
> Fyi: To my knowledge, the examples will be split into separate PRs and will undergo some changes

Yes, those examples will undergo some changes.

**Comment by peacekeeper** [comment]
So this means that by now, compatibility of SD-JWT VC with VCDM has been completely abandoned, right? At this point, we're faced not only with different proof formats, but also completely different data models.

**Comment by danielfett** [comment]
As discussed, separate PRs will be created for the examples.

**Pull request closed without merging**